### ISSUE TITLE
SA1513 Triggers in multiline string interpolation expressions

### ISSUE THREAD

**Issue by vgriph** [DotNetAnalyzers/StyleCopAnalyzers#2947]
Version: 1.1.118
When using multi-line string with interpolation, and breaking the string in an interpolation, the rule SA1513 is triggered for the brace ending the multi-line interpolation:
```cs
                    Log($@"Argument: {
                        (string.IsNullOrEmpty(argument) ? "None" : argument)} supplied");
```
This is maybe not a desired way of breaking the argument across lines, but there is no way to satisfy the rule that doesn't introduce a new-line in the final string.

**Comment by sharwell** [comment]
I believe the preferred approach is to extract the placeholder to a local variable.
